airflow-core/tests/unit/api_fastapi/core_api/routes/public/test_backfills.py

```diff
@@ -92,10 +93,16 @@ def to_iso(val):
 class TestBackfillEndpoint:
     @provide_session
     def _create_dag_models(self, *, count=1, dag_id_prefix="TEST_DAG", is_paused=False, session=None):
+        bundle_name = "dags-folder"
+        orm_dag_bundle = DagBundleModel(name=bundle_name)
+        session.merge(orm_dag_bundle)
+        session.flush()
```

Comment:
We should also add `clear_db_dag_bundles` to cleanup.

Reply:
I see. Did not know this method exists. All tests updated.